tracy-client/src/frame.rs

```diff
@@ -114,6 +114,30 @@ impl Client {
         }
         Frame(self.clone(), name)
     }
+
+    /// Emits an image of the current frame.
+    ///
+    /// The following restrictions apply:
+    /// - The image must be in RGBA format.
+    /// - The image width and height must be divisible by four.
+    /// - The image data must be less than 256 KB, and should be as small as
+    ///   possible.
+    ///
+    /// The offset indicates how many frames in the past the image was captured.
+    /// For example, an offset of 1 would associate the image with the previous
+    /// call to [frame_mark].
```

Comment:
```suggestion
    /// call to [`frame_mark`](Client::frame_mark).
```

probably...

Reply:
The link works without this change :shrug:

Reply:
Oh, I see, this is actually different.